clients/cli/src/register.rs

```diff
@@ -53,33 +53,53 @@ pub async fn register_user(
     }
 
     // Check if the wallet address is already registered with the orchestrator.
+
     if let Ok(user_id) = orchestrator.get_user(wallet_address).await {
-        print_cmd_info!(
-            "Wallet address is already registered with user ID.",
-            "User ID: {}, Wallet Address: {}",
-            wallet_address,
-            user_id
-        );
-        let config = Config::new(
-            user_id,
-            wallet_address.to_string(),
-            String::new(), // node_id is empty for now
-            orchestrator.environment().clone(),
-        );
-        // Save the configuration file with the user ID and wallet address.
-        config
-            .save(config_path)
-            .map_err(|e| handle_cmd_error!(e, "Failed to save config."))?;
+    
+    if !user_id.is_empty() && uuid::Uuid::parse_str(&user_id).is_err() {
```

Comment:
If this is intended to be a nested if statement, it should be indented. I'm not sure this is worth checking though as orchestrator always returns valid user ids